jina/logging/logger.py

```diff
@@ -82,7 +82,15 @@ class SysLogHandlerWrapper(logging.handlers.SysLogHandler):
 class JinaLogger:
     supported = {'FileHandler', 'StreamHandler', 'SysLogHandler', 'FluentHandler'}
 
-    def __init__(self, context: str, log_config: Optional[str] = None, **kwargs):
+    def __init__(self, context: str, log_config: Optional[str] = None, group_id: Optional[str] = None, **kwargs):
```

Comment:
I don't see how this `group_id` is related to `id` in `m.get('pod_id', None)`

Reply:
We can change the name, but the idea is to allow the fluentd to group the logs by an identifier (which could be pod_id or flow_id) provided by the jinad. This way once the dashboard requests specific logs to jinad it will stream by reading from these folders.